components/camel-azure/camel-azure-cosmosdb/src/main/java/org/apache/camel/component/azure/cosmosdb/CosmosDbConfiguration.java

```diff
@@ -93,6 +92,9 @@ public class CosmosDbConfiguration implements Cloneable {
     private CosmosQueryRequestOptions queryRequestOptions;
     @UriParam(label = "producer", defaultValue = "listDatabases")
     private CosmosDbOperationsDefinition operation = CosmosDbOperationsDefinition.listDatabases;
+    @UriParam(label = "common", enums = "SHARED_ACCOUNT_KEY,AZURE_IDENTITY",
```

Comment:
If this is about login etc then I think label = security ??

Reply:
Yes, it has been marked as common all around the Azure components, so I maintained as it is elsewhere, but since I'm reviewing the components, I'll modify accordingly.